docs/contrib/how_to/add_new_annotations_and_extracted_docs.rst

```diff
@@ -0,0 +1,33 @@
+**********************************************
+How to: Add new annotations and extracted docs
+**********************************************
+
+.. contents::
+   :depth: 1
+   :local:
+
+Example annotations and automated docs
+======================================
+
+Annotations are a great way to keep documentation close to the code, but also be able to extract into documentation to be published on Readthedocs.
+
+As an example, here are the `feature toggle annotations in the edx-platform codebase`_, and here is the published `Readthedocs document with the edx-platform feature toggles`_.
```

Comment:
Nitpick: I don't like the 'here is/are...' constructions of this sentence, and I think something like:

"As an example, we have added feature toggle annotations to the edx-platform codebase. A Readthedocs document with the edx-platform feature toggles has been generated from them."